packages/terra-date-input/src/DateInput.jsx

```diff
@@ -570,6 +582,8 @@ class DateInput extends React.Component {
         autoComplete="off"
         disabled={this.props.disabled}
         isInvalid={this.props.isInvalid}
+        isIncomplete={this.props.isIncomplete}
+        required={this.props.required}
       />
```

Comment:
The `isIncomplete` and `required` props should be extracted out of the customProps in the [render](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-input/src/DateInput.jsx#L619) function.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/923/commits/b43c2261fa209a9bbca6d8202315a4cd7754df28?file-filters%5B%5D=.jsx#diff-9620f8593dd0f7ccffbf6a626cc100f9R627